Source/WebCore/html/canvas/CanvasRenderingContext2DBase.cpp

```diff
@@ -1934,7 +1935,9 @@ ExceptionOr<Ref<CanvasGradient>> CanvasRenderingContext2DBase::createConicGradie
     if (!std::isfinite(angleInRadians) || !std::isfinite(x) || !std::isfinite(y))
         return Exception { NotSupportedError };
 
-    return CanvasGradient::create(FloatPoint(x, y), angleInRadians, *this);
+    // Angle starts from x-axis for consistency within canvas methods https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-createconicgradientΩ
```

Comment:
Nit: Comments should be full sentences, so I would rewrite this as:

`// Angle starts from x-axis for consistency within canvas methods. See https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-createconicgradient`

The URL also has an unexpected `Ω` at the end.